examples/pipelines/windows-efi-installer/windows11-installer-pipelinerun.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: windows11-installer-run-
+spec:
+  params:
+    - name: winImageDownloadURL
+      value: >-
+       <INSERT_WINDOWS_ISO_URL>
```

Comment:
Multiline string needed? I would keep it in sync with `windows10-installer-pipelinerun.yaml`.